index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+  <head>
+	<meta charset="utf-8">
+	<title>Michelle's Cookbook!</title>
+    <link rel="Stylesheet" href="index.css">
+  </head>
+  <body>
+      <a href="/Users/michelletan/bootcamp-project-2021/aboutme.html" target="_blank">About Me</a>
+      <p id="title">Michelle's Cookbook!</p>
+	  <p id="info">Hey! I'm Michelle and I love spending my free time cooking and experimenting with new recipes. Here you will find some of my favorite recipes of food I love to make!</p>
+      <br />
+      <h1>Recipes:</h1>
+      <a href="/Users/michelletan/bootcamp-project-2021/pasta.html" target="_blank">Gigi Hadid Pasta</a>
```

Comment:
So here you have a common error that most people ran into. in you < a > tags, you are using an absolute file path to link between your pages. This works fine on you computer, but does not work on any other machines, so your links are broken on my end.
To fix this you can just use the relative file path, which in this case would just be the name of the file (ex: pasta.html)
If you have questions I'm happy to help with this!

Reply:
Hi Ben, thank you so much for your input and your response! I fixed the file paths, let me know if it works! Also, I made a mistake last time committing my first milestone files only, but I just committed the second milestone files. Let me know if you can see them :)